src/Components/Authentication/Footer.tsx

```diff
@@ -11,15 +11,9 @@ const FooterContainer = styled.div`
   display: ${(props: Props) => (props.inline ? "inline" : "flex")};
   flex-direction: column;
 `
-
+// TODO: Re-enable twitter link when @artsy/passport is twitter-functional
```

Comment:
I created a ticket to track removing twitter altogether as we discussed.
https://artsyproduct.atlassian.net/browse/GROW-794

Maybe let's update this TODO with that instead